cms/djangoapps/models/settings/course_details.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, org, course_id, run):
         self.syllabus = None  # a pdf file asset
         self.short_description = ""
         self.overview = ""  # html to render as the overview
-        self.about_sidebar_html = "" # html to render as the about_sidebar_html
+        self.about_sidebar_html = ""  # html to render as the about_sidebar_html
```

Comment:
This comment is superfluous; it tells us nothing that the code doesn't already. Let's just:
- remove this comment entirely
- change double-quotes to single-quotes